src/test/java/org/taf/FilterTest.java

```diff
@@ -0,0 +1,63 @@
+package org.taf;
+
+import com.epam.reportportal.junit5.ReportPortalExtension;
+import io.github.bonigarcia.seljup.SeleniumJupiter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.taf.pages.LoginPage;
+import org.taf.pages.MainPage;
+import org.taf.utils.RepositoryParser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(SeleniumJupiter.class)
```

Comment:
Does it work without this line?

Reply:
It works locally for me. More information you can find there https://bonigarcia.dev/selenium-jupiter/

Reply:
The questions are still actual:
1) Does it works without this annotation?
2) What is its purpose here?

AFAIK, this annotation is intended to manage driver lifecycle, but you have this management logic in @BeforeEach and @AfterEach methods.

Reply:
I decided to use this annotation and update other part of the code.